src/EasyAbp.Abp.PhoneNumberLogin.Application/EasyAbp/Abp/PhoneNumberLogin/Account/PhoneNumberLoginAccountAppService.cs

```diff
@@ -276,7 +296,19 @@ protected virtual async Task<string> GenerateCodeAsync(string phoneNumber, Verif
             switch (type)
             {
                 case VerificationCodeType.ResetPassword:
-                    code = await _identityUserManager.GeneratePasswordResetTokenAsync(identityUser);
+                    if (identityUser == null)
+                        throw new UserFriendlyException(_localizer["InvalidPhoneNumber"]);
```

Comment:
What about `throw new InvalidVerificationCodeException();`?

Reply:
if u  used twofactor to generate , u need use identityuser not null , identityuser come from phonenumber in database. database doesn`t have this phonenumber u cant resetpassword.